src/service/url-replacements-impl.js

```diff
@@ -52,7 +54,7 @@ const ORIGINAL_VALUE_PROPERTY = 'amp-original-value';
  * @param {*} val
  * @return {string}
  */
-function encodeValue(val) {
+export function encodeValue(val) {
```

Comment:
is this used outside?

Reply:
I was using this at first, but deleted in a refactor, and forgot to clean this up. Export is removed.